docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
An alternative to the config generator would simply be the ability to specify multiple resources on one command line:
```
kubectl create --app=guestbook service rc count:3 podtemplate containers:'[{image:brendanburns/redis-slave}]'
```